src/contexts/DpiTokenMarketData/DpiTokenMarketDataProvider.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from 'react'
+import MarketDataContext from './DpiTokenMarketDataContext'
+import { coingeckoDpiId } from 'constants/coingeckoIds'
+import { fetchHistoricalTokenMarketData } from 'utils/coingeckoApi'
+
+interface HistoricalMarketDataProvider {
+  // coingeckoId: string
+	daysOfHistory: number
```

Comment:
I'm surprised this didn't get picked up by the auto linter? Let me know if it isn't working for you. You should be able to make a small change to this file, commit it and it should fix this kind of stuff.